topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/slides.html

```diff
@@ -0,0 +1,463 @@
+
+.image-60[ ![dim red](../../images/scrna-casestudy-monocle/dim_red_methods.png) ]
+
+---
+
+### Extensions of trajectory inference: **RNA velocity methods**
+.footnote[[Deconinck et al. Recent advances in trajectory inference from single-cell omics data](https://doi.org/10.1016/j.coisb.2021.05.005.)]
+--
+
+- RNA velocity methods estimate the future state of a single cell captured in a static snapshot by looking at the ratios between spliced mRNA, unspliced mRNA, and mRNA degradation. 
+
+--
+
+- scVelo and CellRank use this extra velocity information to construct a directed k-nearest neighbor graph, as a starting step for the TI method. 
+
+--
+
+- This has the advantage that root cell specification is not necessary, and adds directional information to the trajectory.
+
+--
+
+.image-25[ ![velocity](../../images/scrna-casestudy-monocle/velocity.png) ]
+
+---
+### Assumptions
+--
+
+- It would be quite difficult to analyse a sample every few seconds to see how the cells are changing. Therefore, we assume that snapshot data encompass all naïve, intermediate, and mature cell states with sufficient sampling coverage to allow the reconstruction of differentiation trajectories.    
+[Sagar and Grün, 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115822/)
+
+--
+
+- As different TI methods make different assumptions about the data, a first choice to make is based on which biological process is to be expected: **not all TI methods are designed to infer all kinds of biological processes.**  
+[Deconinck et al., 2021](https://doi.org/10.1016/j.coisb.2021.05.005)
+
+---
+### Why different TI methods are designed to infer different kinds of biological processes?
+--
+
+Look at the pipeline presented by [Robrecht Cannoodt et al., 2016](https://onlinelibrary.wiley.com/doi/10.1002/eji.201646347)
+
+--
+
+.image-100[ ![pipeline](../../images/scrna-casestudy-monocle/7_methods.jpg) ]        
+--
+
+There are multiple methods using particular algorithms, or even their combinations, so you must consider which one would be best for analysing your sample.
+
+---
+
+### When analysing your data, consider the following: 
+--
+
+.pull-left[  
+  
+
+- Tissue from which the cells come from   
+
+- Branching points  
+
+- Supervised and unsupervised learning  
+
+- Format of the data  
+
+- Number of cells and features  
+
+- Computing power & running time   
+
+]
+
+--
+
+.pull-right[
+    
+ 
+To help you evaluate which method would work best for your data, check out this [**awesome comparision site - dynguidelines**](https://zouter.shinyapps.io/server/), a part of a larger set of open packages for doing and interpreting trajectories called [the dynverse](https://github.com/dynverse/dynverse).     
+  
+![zouter](../../images/scrna-casestudy-monocle/zouter.jpg) 
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

(this seems to apply to the rest of the images as well in this deck)

Reply:
All images, both in the slide deck and tutorial, have now detailed alt text added to make those resources as accessible as possible!

Reply:
thank you so much!!! perfect!